### PR TITLE
feat(cli): add support for jenkinsfile template in sl scaffold

### LINKED ISSUE #2149: @sourceloop/cli:add updated eslintrc.js file and template support for jenkins
**Describe the bug**
1.update eslintrc.js file with 'no-await-in-loop' for scaffold and microservices
2.add support foe jenkinsfile template in cli while scaffolding

### PR DESCRIPTION
**Pull request by Surbhi-sharma1**
add support for jenkinsfile template in sl scaffold

GH-2149

## Description

add support for jenkinsfile template in sl scaffold

Fixes #2149 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] Performed a self-review of my own code
- [x] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/cli/src/command-base.ts
Comment (yeshamavani) [inline packages/cli/src/command-base.ts:1]: why is this added to base command and not the scaffold command file

Comment (Surbhi-sharma1) [reply packages/cli/src/command-base.ts:1]: was getting issue with helmprompt option that needs to be prompted while adding this in scaffold file .
